### DISCUSSION TITLE
Βαθμός Εξεταστικής Σεπτέμβριου 2021

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1528] [Announcements]
Καλησπέρα, ονομάζομαι Μάριος Κωνσταντίνου – Π2015002 και βαθμολογήθηκα με 3 σε αυτή την εξεταστική.

Θα μπορούσατε να ξανά δείτε το branch μου, ίσος έγινε κάποιο λάθος; Είχα κάνει όλες τις ασκήσεις γραμμής εντολών και προγραμματισμού και ένα κομμάτι του συμμετοχικού υλικού.

**Comment by c15pati** [comment]
@mariosconsta  Δυστυχώς το συμμετοχικό υλικό σου, δεν ήταν ούτε καν κοντά στην βάση - για αυτό το λόγο, και ο βαθμός.
